packages/management/addons-manager/bundle/vendir.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: vendir.k14s.io/v1alpha1
+kind: Config
+minimumRequiredVersion: 0.12.0
+directories:
+  - path: 1.5.0/bundle/config
+    contents:
+      - path: upstream
+        manual: {}
+      - path: overlays
+        manual: {}
+      - path: values.yaml
+        manual: {}
+  - path: 1.5.0/upstream-package.yaml
```

Comment:
same here, the version needs to be removed and path should be `package.yaml`, can you also fix other similar kind of issues in this file?

Reply:
@yharish991 updated the vendir.yml and updated the path error, removed version directory and corrected file name